sound/soc/sof/intel/hda.h

```diff
@@ -464,8 +464,8 @@ int hda_dsp_stream_spib_config(struct snd_sof_dev *sdev,
  * DSP IPC Operations.
  */
 int hda_dsp_ipc_is_ready(struct snd_sof_dev *sdev);
-int hda_dsp_ipc_send_msg(struct snd_sof_dev *sdev,
-			 struct snd_sof_ipc_msg *msg);
+void hda_dsp_ipc_send_msg(struct snd_sof_dev *sdev,
+			  struct snd_sof_ipc_msg *msg);
```

Comment:
Humm. I can think of devices where IPC can fail and e.g. needs to be retried. It's not because we don't need this at Intel that we should remove this capability?

Reply:
> Humm. I can think of devices where IPC can fail and e.g. needs to be retried. It's not because we don't need this at Intel that we should remove this capability?

@plbossart in that case I will leave the original definitions and check for failures before moving the message from the list.